File: 1672. Richest Customer Wealth/main.py
from typing import List


class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_wealth = 0
        for customer in accounts:
            max_wealth = max(max_wealth, sum(customer))

        return max_wealth


class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        return max(sum(customer) for customer in accounts)

File: 1672. Richest Customer Wealth/main.rs
impl Solution {
    pub fn maximum_wealth(accounts: Vec<Vec<i32>>) -> i32 {
        let mut max_wealth: i32 = 0;
        for customer in accounts {
            max_wealth = max_wealth.max(customer.iter().sum());
        }

        max_wealth
    }
}

impl Solution {
    pub fn maximum_wealth(accounts: Vec<Vec<i32>>) -> i32 {
        accounts
            .iter()
            .map(|customer| customer.iter().sum())
            .max()
            .unwrap()
    }
}
